### ISSUE TITLE
Unexpected behavior of .containEql when target object has keys not in source object

### ISSUE THREAD

**Issue by apadmarao** [tj/should.js#189]
Hello,

```
sourceObject.containEql(targetObject) 
```

throws a TypeError when targetObject has keys not in the sourceObject.  We expect an AssertionError to be thrown instead.

```
{}.should.containEql({foo: 'foo'}) // throws TypeError: Cannot read property 'should' of undefined
```

I am happy to submit a PR for this, if we agree on the expected behavior.

**Comment by btd** [comment]
Yes, looks like a bug, it should throw AssertionError.

**Comment by btd** [comment]
Fixed in 3.1.3
